sqlite/ext/comdb2/schistory.c

```diff
@@ -99,9 +99,7 @@ static int get_status(void **data, int *npoints)
         char str[22];
         sprintf(str, "%0#16"PRIx64, hist[i].seed);
         sc_hist_ents[i].seed = strdup(str);
-
-        if (hist[i].errstr)
-            sc_hist_ents[i].error = strdup(hist[i].errstr);
+        sc_hist_ents[i].error = strdup(hist[i].errstr);
```

Comment:
won't this crash if errstr is null?

Reply:
It won't be null because it is not a pointer - clang was (correctly) warning that test is always true.